app/styles/cards.module.css

```diff
@@ -166,7 +166,7 @@
   }
 
   .card {
-    padding: 30px 10px;
+    padding: 30px 20px;
```

Comment:
do not change the general stylings, this can break the UI of other component which is using this class